Alignment/OfflineValidation/python/TkAlAllInOneTool/betterConfigParser.py

```diff
@@ -153,6 +153,8 @@ def getGeneral( self ):
         if not self.has_option(internal_section, "workdir"):
             self.set(internal_section, "workdir", "/tmp/$USER")
         general["workdir"] = self.get(internal_section, "workdir")
+        for dir in "workdir", "datadir", "logdir", "eosdir":
+            general[dir] = os.path.expandvars(general[dir])
         return general
```

Comment:
@hroskes right, in this case it does not matter.
And yes, please fix it in the next iteration.